src/ResourceManager/Compute/Commands.Compute/Common/ComputeClientBaseCmdlet.cs

```diff
@@ -40,10 +43,30 @@ public ComputeClient ComputeClient
 
             set { computeClient = value; }
         }
+
         public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
             ComputeAutoMapperProfile.Initialize();
         }
+
+        protected void ExecuteClientAction(Action action)
+        {
+            try
+            {
+                try
```

Comment:
no need to make try twice. Just make sure you catch CloudException before general Exception.